rocket_sqlite_oracle/src/main.rs

```diff
@@ -0,0 +1,252 @@
+use anyhow::{Context, Result};
+use rocket::fairing::AdHoc;
+use rocket::serde::json::Json;
+use rocket::serde::Serialize;
+use rocket::{fairing, get, routes, tokio, Build, Rocket, State};
+use rocket_db_pools::{sqlx, Connection, Database};
+use rocket_sqlite_oracle::kraken;
+use secp256k1::rand::thread_rng;
+use secp256k1::{PublicKey, SecretKey, Signature};
+use secp256k1_zkp::SECP256K1;
+use std::convert::{TryFrom, TryInto};
+use std::path::{Path, PathBuf};
+use structopt::StructOpt;
+use tokio::io::AsyncWriteExt;
+use url::Url;
+
+use baru::oracle::Message;
+
+#[derive(Database)]
+#[database("oracle_signatures")]
+struct ExampleDB(sqlx::SqlitePool);
+
+#[derive(Serialize)]
+struct USDTickerResponse {
+    timestamp: u64,
+    ask_price: u64,
+    bid_price: u64,
+    signature: Signature,
```

Comment:
:wrench: I don't think we should return the `bid_price` since we're not signing it.

Reply:
Table + endpoints changed to distinguish between bid and ask prices as well as the associated signatures, so the GET requests are now like:
```
http://127.0.0.1:8000/ask/price/
http://127.0.0.1:8000/bid/price/
```

close price values have been discarded as they have no conceivable use to us.